sdk/core/azure-core-http-jdk-httpclient/src/test/java/com/azure/core/http/jdk/httpclient/JdkHttpClientTests.java

```diff
@@ -336,7 +336,8 @@ public void testConcurrentRequests() {
         ParallelFlux<byte[]> responses = Flux.range(1, numRequests)
             .parallel()
             .runOn(Schedulers.boundedElastic())
-            .flatMap(ignored -> doRequest(client, "/long").flatMap(HttpResponse::getBodyAsByteArray));
+            .flatMap(ignored -> doRequest(client, "/long"))
+            .flatMap(response -> Mono.using(() -> response, HttpResponse::getBodyAsByteArray, HttpResponse::close));
```

Comment:
are there any other concurrency tests in SDKs our team owns that we should make the same test fix?

Reply:
There aren't, or they're using types that `HttpResponse` is converted into which doesn't run into this issue.